### PR TITLE
docs: add docs for @server(responseHeaders)

### LINKED ISSUE #46: Document `@server(responseHeaders)`

### PR DESCRIPTION
**Pull request by rajdip-b**
Fixes #46 

/claim #46

### CODE REVIEW

File: docs/guides/operators.md [hunk 1]
```diff
@@ -147,6 +147,15 @@ schema @server(responseValidation: true) {
 Disabling this setting will offer major performance improvements, but at the potential expense of data.
 :::
 
+### responseHeaders
+
+The `responseHeader` is an array of key-value pairs. These headers are added to the response of every request made to the server. This can be useful for adding headers like `Access-Control-Allow-Origin` to allow cross-origin requests, or some 
+additional headers like `X-Api-Key` to be used by the downstream services.
```
Comment (tusharmath) [inline docs/guides/operators.md:153]: Not sure if the x api key header is a good example.

**Review by tusharmath — CHANGES_REQUESTED** [review]

File: docs/guides/operators.md (hunk 1 above)
Comment (rajdip-b) [reply docs/guides/operators.md:153]: Should I replace it with something else or just remove it?

Comment (rajdip-b) [reply docs/guides/operators.md]: @tusharmath I have updated the example.

**Comment by rajdip-b** [comment]
Hey @tusharmath, can you give it a review? I made some fixes since the lint was crashing.

**Comment by tusharmath** [comment]
@rajdip-b can you run prettier and push once.

**Comment by rajdip-b** [comment]
> @rajdip-b can you run prettier and push once.

I didn't see any prettier scripts. Should I run `npx prettier --fix` ?

**Comment by rajdip-b** [comment]
Hey @tusharmath, I have fixed the prettier issues.

**Comment by rajdip-b** [comment]
Hey @tusharmath, any updates or features that you want me to make to this PR?

**Review by tusharmath — APPROVED** [review]

**Pull request merged**